ui-lib/src/main/res/ui/account/values-es/strings.xml

```diff
@@ -19,12 +19,11 @@
     <plurals name="account_history_item_message">
         <item quantity="one">Mensaje</item>
         <item quantity="other">Mensajes</item>
-        <item quantity="many">Mensajes</item>
```

Comment:
Should we keep the `many`? I see this warning in Android Studio now: `For locale "es" (Spanish) the following quantity should also be defined: many`